groceries_app/urls.py

```diff
@@ -1,9 +1,9 @@
 from django.urls import path
-from .views import HomeView, AboutView, RecipesView, PlannerView
+from .views import HomeView, AboutView, RecipesView, MealChoiceView
 
 urlpatterns = [
     path('', HomeView.as_view(), name='home'),
     path('about', AboutView.as_view(), name='about'),
     path('recipes', RecipesView.as_view(), name='recipes'),
-    path('planner', PlannerView.as_view(), name='planner')
+    path('meal-choice', MealChoiceView.as_view(), name='meal-choice')
```

Comment:
Is it not called grocery planner anymore..?

Reply:
With the mindset of simplicity, I decided to change the url to meal-choice and each part of the planner would have it's own unique url. As you bring this up, I think it would make more sense to structure this as 'planner/meal-choice' and future pages would be 'planner/xyz'. I will make this change.